dashboard/test/models/script_test.rb

```diff
@@ -1534,9 +1549,41 @@ def populate_cache_and_disconnect_db
     assert_equal true, script.hidden
   end
 
+  test 'has pilot access' do
+    teacher = create :teacher
+    levelbuilder = create :levelbuilder
+
+    script = create :script
+    assert_equal false, script.pilot?
+    assert_equal false, script.has_pilot_access?
+    assert_equal false, script.has_pilot_access?(teacher)
+    assert_equal false, script.has_pilot_access?(levelbuilder)
+
+    # for now, only levelbuilders have pilot script access.
+    script.pilot_experiment = 'my-experiment'
+    assert_equal true, script.pilot?
+    assert_equal false, script.has_pilot_access?
+    assert_equal false, script.has_pilot_access?(teacher)
+    assert_equal true, script.has_pilot_access?(levelbuilder)
+  end
+
+  test 'has any pilot access' do
+    teacher = create :teacher
+    levelbuilder = create :levelbuilder
+
+    # for now, only levelbuilders have any pilot access.
+    assert_equal false, Script.has_any_pilot_access?
+    assert_equal false, Script.has_any_pilot_access?(teacher)
+    assert_equal true, Script.has_any_pilot_access?(levelbuilder)
+  end
+
   private
 
   def has_hidden_script?(scripts)
     scripts.any?(&:hidden)
   end
+
+  def has_pilot_script?(scripts)
+    scripts.any?(&:pilot?)
+  end
```

Comment:
Nit: Your method implementation is shorter than the method name!  Might be better without this helper.